mx.sulong/suite.py

```diff
@@ -357,11 +357,25 @@
       "license" : "BSD-new",
     },
 
+    "com.oracle.truffle.llvm.assemblerX86" : {
```

Comment:
If this is 64 bit x86, then I think Oracle usually uses the name 'AMD64'. Might be good to use that for consistency?